flatdata-generator/flatdata/generator/tree/builder.py

```diff
@@ -152,13 +152,31 @@ def _compute_structure_sizes(root):
             offset += int(field.type.width)
         struct.size_in_bits = offset
 
+def _check_ranges(root):
+    # First check that names are unique
+    for field in root.iterate(nodes.Field):
+        name = field.range_with_next
+        if not name:
+            continue
+        for sibling in field.parent.fields:
+            if sibling.name == name:
+                raise InvalidRangeName(name = name)
+
+    # Now check that structs with ranges are only used in vectors
+    for reference in root.iterate(StructureReference):
+        if isinstance(reference.node, nodes.Structure) and reference.node.has_range_with_next  \
```

Comment:
wrap the condition with ( ), then you won't need \

Reply:
Wasn't sure what python preferred syntax was, will do

Reply:
Preferred is to use parentheses. I don't remember though where I have this rule from, can't find a reference.